beacon-chain/execution/service.go

```diff
@@ -696,12 +705,16 @@ func (s *Service) determineEarliestVotingBlock(ctx context.Context, followBlock
 		}
 		return earliestBlk, nil
 	}
+	if s.latestEth1Data.BlockHeight < params.BeaconConfig().Eth1FollowDistance {
```

Comment:
this should only come into play in testnets - when the chain hasn't advanced past the follow distance, we don't want to consider any block past the genesis block.

Reply:
TODO: code comment about this.